code/service/src/main/scala/com/namely/chiefofstate/SchemasUtil.scala

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2020 Namely Inc.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.namely.chiefofstate
+
+import com.typesafe.config.Config
+import org.slf4j.{Logger, LoggerFactory}
+import slick.basic.DatabaseConfig
+import slick.jdbc.PostgresProfile
+
+import java.sql.{Connection, Statement}
+
+/**
+ * Utility class to create the necessary schemas for the write side
+ */
+object SchemasUtil {
+  final val log: Logger = LoggerFactory.getLogger(getClass)
+
+  /**
+   * journalTableStatement returns the journal ddl
+   *
+   * @param table the table name
+   * @return the sql statement
+   */
+  private def journalTableStatement(): Seq[String] = {
+    Seq(
+      s"""
+        CREATE TABLE IF NOT EXISTS event_journal(
```

Comment:
should we omit new new table DDL's for now from the schema util? Does akka 5.0.0 work without them?

Reply:
Since we are not cutting any release yet we can leave them because that is the way going forward. Otherwise I will have to add that backward compatibility config which we don't want rather providing the migration tool.